README.md

```diff
@@ -75,6 +75,7 @@ the ruleset.
 - Ten Ten
 - Tinder
 - Tokopedia
+- [BazelPods](https://github.com/sergeykhliustin/BazelPods)
```

Comment:
Can you move this up to be alphabetical? Also, can you remove the link here, and add a permalink to https://github.com/sergeykhliustin/BazelPods/blob/main/xcodeproj/BUILD in the examples README?

Reply:
done